easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-goolfc-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,38 @@
+name = 'TensorFlow'
+version = '1.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'goolfc', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
+    'TensorFlow-%(version)s_dont_expand_cudnn_path.patch',
+    'TensorFlow-%(version)s_dont_expand_cuda_path.patch',
+]
+checksums = [
+    '0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8',  # v1.5.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.5.0_swig-env.patch
+    '93ba981296cf4e0d2411fbf6380dd325fa41ebff5d82b02ffe7747f733bca579',  # TensorFlow-1.5.0_dont_expand_cudnn_path.patch
+    '3dcefd56748a6e1bd0cfb1e08e0b33d5d47dbaed7cb706dd7e6842413faf54a7',  # TensorFlow-1.5.0_dont_expand_cuda_path.patch
+]
+
+builddependencies = [
+    ('Bazel', '0.7.0'),
+    ('wheel', '0.30.0', versionsuffix),
+]
+dependencies = [
+    ('Python', '3.6.3'),
+    ('cuDNN', '7.0.5.15'),
+]
+
+with_mkl_dnn = False
```

Comment:
@akesandgren Isn't this something that could/should be handled by the `TensorFlow` easyblock? Basically `if cuDNN then disable mkl_dnn`?

Reply:
Yes, it probably should be done in the easyblock. (As I think i said in the easyblock PR?)
But since it wasn't done I added it here for the time being.

Reply:
I updated the `TensorFlow` to do this automagically now, see https://github.com/easybuilders/easybuild-easyblocks/pull/1287/commits/960e97c889c14b78a90c4b8ab384eeeeefa4d52e, so you can drop this line.